client/my-sites/domains/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -134,17 +163,17 @@ class EditContactInfoFormCard extends React.Component {
 	}
 
 	renderBackupEmail() {
-		const currentEmail = this.props.contactInformation.email,
+		const { email } = this.getContactFormFieldValues(),
 			wpcomEmail = this.props.currentUser.email,
 			strong = <strong />;
 
 		return (
 			<p>
 				{ this.props.translate(
-					'If you don’t have access to {{strong}}%(currentEmail)s{{/strong}}, ' +
+					'If you don’t have access to {{strong}}%(email)s{{/strong}}, ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'If you don’t have access to {{strong}}%(currentEmail)s{{/strong}}, we will also email you at {{strong}}%(wpcomEmail)s{{/strong}}, as backup.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).